src/lib/flows/claim-project-flow/steps/enter-git-url/enter-git-url.ts

```diff
@@ -0,0 +1,50 @@
+import type { State } from '../../claim-project-flow';
+import unreachable from '$lib/utils/unreachable';
+import { getChangedTemplate } from '../add-ethereum-address/drips-json-template';
+import GitHub from '$lib/utils/github/GitHub';
+import { get, type Writable } from 'svelte/store';
+import walletStore from '$lib/stores/wallet/wallet.store';
+import { Octokit } from '@octokit/rest';
+
+const octokit = new Octokit();
+const github = new GitHub(octokit);
+
+export async function loadingFundingInfo(context: Writable<State>): Promise<void> {
```

Comment:
minor nitpick but why `loading` and not `load`?

Reply:
Oops, no reason. Updated!